hkmc2/shared/src/main/scala/hkmc2/codegen/js/JSBuilder.scala

```diff
@@ -251,14 +251,14 @@ class JSBuilder(using TL, State, Ctx) extends CodeBuilder:
             
             def mkMethods(mtds: Ls[FunDefn], mtdPrefix: Str)(using Scope): Document =
               mtds.map:
-                case td @ FunDefn(_, _, ps :: pss, bod) =>
+                case td @ FunDefn(_, _, _, ps :: pss, bod) =>
```

Comment:
These are all anti-patterns (tons of underscore pattern args) that we should take this opportunity to fix
```suggestion
                case td @ FunDefn(params = ps :: pss, body = bod) =>
```